packages/ui/src/ui-component/dialog/ManageScrapedLinksDialog.js

```diff
@@ -0,0 +1,184 @@
+                            sx={{ borderRadius: '12px', mt: 1, display: 'flex', flexShrink: 0 }}
+                            size='small'
+                            variant='contained'
+                            onClick={handleFetchLinks}
+                        >
+                            Fetch Links
+                        </Button>
+                    </Stack>
+                </Box>
+                <Typography sx={{ mb: 2, fontWeight: 500 }}>Scraped Links</Typography>
+                {selectedLinks.length > 0 ? (
+                    <PerfectScrollbar
+                        style={{
+                            height: '100%',
+                            maxHeight: '320px',
+                            overflowX: 'hidden',
+                            display: 'flex',
+                            flexDirection: 'column',
+                            gap: 4
+                        }}
+                    >
+                        {selectedLinks.map((link, index) => (
+                            <div key={index} style={{ display: 'flex', width: '100%' }}>
+                                <Box sx={{ display: 'flex', width: '100%' }}>
+                                    <OutlinedInput
+                                        sx={{ width: '100%' }}
+                                        key={index}
+                                        type='text'
+                                        onChange={(e) => handleChangeLink(index, e)}
+                                        size='small'
+                                        value={link}
+                                        name={`link_${index}`}
+                                    />
+                                </Box>
+                                <Box sx={{ width: 'auto', flexGrow: 1 }}>
+                                    <IconButton
+                                        sx={{ height: 30, width: 30 }}
+                                        size='small'
+                                        color='error'
+                                        onClick={() => handleRemoveLink(index)}
+                                        edge='end'
+                                    >
+                                        <IconTrash />
+                                    </IconButton>
+                                </Box>
+                            </div>
+                        ))}
+                    </PerfectScrollbar>
+                ) : (
+                    <>
+                        {loading && <BackdropLoader open={loading} />}
+                        <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+                            <Typography sx={{ my: 2 }}>Links scraped from the URL will appear here</Typography>
+                        </div>
+                    </>
+                )}
+            </DialogContent>
+            <DialogActions>
+                <Button onClick={onCancel}>Cancel</Button>
```

Comment:
dialogProps.cancelButtonName ?

Reply:
I think it's fine for it to be hardcoded here. We don't really gain anything by making it a prop since the component has a specific use and not a generalized one like `Dialog` or `Button`.